packages/cli/README.md

```diff
@@ -0,0 +1,55 @@
+# auto CLI
```

Comment:
Is this file generated in some way? Seems like it'd be easy to get out of date.

Reply:
each command docs has the --help output displayed too. none of this is automated. would be great if there was a script to just update it.

hopefully people catch on that you can call -h for any command + the whole cli